test/WebSites/HtmlGenerationWebSite/Views/HtmlGeneration_Home/Link.cshtml

```diff
@@ -46,6 +46,20 @@
           asp-fallback-test-property="visibility"
           asp-fallback-test-value="hidden" />
 
+    <!-- Fallback to static href with dynamic attributes -->
+    <link href="~/styles/site.min.css?a=b&c=d" rel="@("stylesheet")" data-extra="@("test")" title='@(@"sylesheet""the"" title")'
```

Comment:
Looks good and new test is running correctly. But is `sylesheet` a typo in the `title` value? Test would be fine w/o those extra letters.